tests/python/gpu/test_profiler_gpu.py

```diff
@@ -23,12 +23,16 @@
 import mxnet as mx
 mx.test_utils.set_default_context(mx.gpu(0))
 
+from mxnet.gluon.block import _block_scope
+
 curr_path = os.path.dirname(os.path.abspath(os.path.expanduser(__file__)))
 sys.path.insert(0, os.path.join(curr_path, '../unittest'))
-from mxnet import profiler
-from mxnet.gluon import nn
-from mxnet.gluon.block import _block_scope
-from test_profiler import enable_profiler
+# We import all tests from ../unittest/test_profiler.py
+# They will be detected by test framework, as long as the current file has a different filename
+from test_profiler import *
+# Test seen to crash pytest worker during development of
+# https://github.com/apache/incubator-mxnet/pull/18694
+del test_aggregate_duplication
```

Comment:
This was working prior to #18704? Could you try if it is really broken now, and if so, how to fix it?

```suggestion
```

Reply:
This was not working before #18704 . I just reverted the changes on that.

Reply:
Thank you for clarifying. @Zha0q1, you added the testcase, do you have an idea why it fails on GPU?

Reply:
Can you share the error message? I just copied out and ran this test on a gpu instance and it did not fail. This test was added in a pr that fixes the operator invocation count bing 2X of the actual number issue. All it does is just create a tensor, do some operations on it, and check if count is correct

Reply:
@Zha0q1 it was skipped as part of https://github.com/apache/incubator-mxnet/pull/18694/commits/60dda9ba884bb35be1ac1a9bd16a2a917073e9ed

I'm not sure about the exact error. Maybe you can just re-enable the test and it will pass.

Reply:
It seems that this line is added by @DickJC123 . Perhaps he can explain why.

Reply:
just this test. I can revisit this after other changes stabilize

Reply:
@leezu @Zha0q1 I tried re-enable these tests and there seems to be no errors.